src/app/shared/local-storage/local-storage.service.ts

```diff
@@ -1,3 +1,5 @@
+import { Injectable } from "@angular/core";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/iver-wharf/wharf-web/pullRequest?prid=7535715)
```suggestion
import { Injectable } from '@angular/core';
```